packages/tupaia-web/src/features/Dashboard/DashboardMenu.tsx

```diff
@@ -61,11 +77,11 @@ export const DashboardMenu = ({
       {activeDashboard && (
         <MenuButton onClick={handleClickListItem} disabled={!hasMultipleDashboards}>
           {activeDashboard?.name}
-          {hasMultipleDashboards && <ArrowDropDownIcon />}
+          {hasMultipleDashboards && <KeyboardArrowDownIcon />}
         </MenuButton>
       )}
 
-      <Menu
+      <ItemButton
```

Comment:
Can we rename this to be more descriptive? `ItemButton` implies one single button, but this is for the whole menu